reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -163,10 +173,20 @@ abstract class TableReporter : Reporter() {
         val scanRecord = ortResult.scanner!!.results
 
         val projectTables = analyzerResult.projects.associate { project ->
+            val projectExclude = ortResult.repository.config.excludes?.findProjectExclude(project)?.let { exclude ->
+                // Only add the project exclude to the model if the whole project is excluded. If only parts of the
+                // project are excluded this information will be stored in the rows of the affected dependencies.
+                exclude.takeIf { it.exclude }
```

Comment:
As a side note, didn't we want to rename `ProjectExclude.exclude` to something more speaking? I'd still prefer something like `isWholeProjectExcluded`, even if it's long.